### ISSUE TITLE
Anomalous slopes from ramp fitting

### ISSUE THREAD

**Issue by hbushouse** [STScI-JWST/jwst#1655]
Julia Duval is reporting that anomalous slope values seen in B7.0 testing have decreased greatly in frequency and severity, but still seeing a few outliers using B7.1 (https://stsci-ins.basecamphq.com/projects/11477312-jwst-pipeline/posts/104444455/comments).

**Comment by hbushouse** [comment]
This may have been fixed by #1596. Do we have the original example data from the user so that we could test using the latest code?

**Comment by hbushouse** [comment]
Latest report from latest code on master:
```
Julia Roman-Duval Thu, 8 Feb at 10:26am
I re-tested the ramp fit with the same data, and the issue has been resolved in the dev version! Case closed, thanks for looking into it.
```
So we can consider this closed. Fixed by #1596.
